frontend/src/TooljetDatabase/Forms/ColumnForm.jsx

```diff
@@ -156,16 +167,18 @@ const ColumnForm = ({ onCreate, onClose, rows }) => {
           <input
             value={defaultValue}
             type="text"
-            placeholder="Enter default value"
+            placeholder={dataType?.value === 'serial' ? 'Auto-generated' : 'Enter default value'}
             className={
-              isNotNull === true && defaultValue.length <= 0 && rows.length > 0 ? 'form-error' : 'form-control'
+              dataType?.value !== 'serial' && isNotNull === true && defaultValue.length <= 0 && rows.length > 0
```

Comment:
can we define these conditions in a const variable and reuse them?

```js
const defaultValueRequired = dataType.value !== 'serial' && isNotNull && isEmpty(defaultValue) && !isEmpty(rows) 
```

Also let us always start to stick with to use `cx` from classnames when we are handling condition for CSS classes